parcels/examples/example_moving_eddies.py

```diff
@@ -117,35 +132,44 @@ def test_moving_eddies_fwdbwd(mode, npart=2):
     pset.execute(method, endtime=0, dt=-dt,
                  output_file=pset.ParticleFile(name="EddyParticlebwd", outputdt=outputdt))
 
-    assert(pset[0].lon > 3.2 and 45.9 < pset[0].lat < 46.1)
-    assert(pset[1].lon > 3.2 and 47.7 < pset[1].lat < 47.9)
-
+    assert np.allclose([p.lon for p in pset], lons)
+    assert np.allclose([p.lat for p in pset], lats)
```

Comment:
Why is this new? It is not related to the other changes.
We were not checking the particle comes back to where it initially started?

Reply:
The locations used to be hardcoded; now we check whether the final locations of the fwd_bwd run are equal to the start locations. So it's not really new; it's just more general